components/grid/grid.html

```diff
@@ -0,0 +1,111 @@
+                        <li><a href="../slider/slider.html">Slider</a></li>
+                        <li><a href="../text-utilities/textUtilities.html">Text Utilities</a></li>
+                        <li><a href="../toast/toast.html">Toast</a></li>
+                    </ul>
+                </div>
+            </div>
+            <!-- documentation -->
+            <div class="documentation-container">
+                <h1>Grid</h1>  
+                <h2>50-50 grid</h2>        
+                <div class="grid col-50-50">
+                    <div class="grid-first-col">
+                        Column 1
+                    </div>
+                    <div class="grid-second-col">
+                        column 2
+                    </div>
+                </div>
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C184%2C195%2C100%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=48px&ph=32px&ln=false&fl=1&fm=Hack&fs=13px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid%2520col-50-50%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522grid-first-col%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520Column%25201%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522grid-second-col%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520column%25202%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                style="max-width: 500px; width: 100%; height: 320px; border:0; margin: 1rem 0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+                </iframe>
+
+                <h2>70-30 grid</h2> 
+                <div class="grid col-70-30">
+                    <div class="grid-first-col">
+                        Column 1
+                    </div>
+                    <div class="grid-second-col">
+                        column 2
+                    </div>
+                </div>
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C184%2C195%2C100%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=48px&ph=32px&ln=false&fl=1&fm=Hack&fs=13px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid%2520col-70-30%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522grid-first-col%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520Column%25201%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522grid-second-col%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520column%25202%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                style="max-width: 500px; width: 100%; height: 320px; border:0; margin: 1rem 0; transform: scale(1); overflow:hidden;"
```

Comment:
avoid inline styles

Reply:
I didn't want to mix carbon iframe CSS with component CSS files and also each iframe has different width and height so that's why I used inline styling